ape_alchemy/provider.py

```diff
@@ -85,6 +98,14 @@ def ws_uri(self) -> str:
         # NOTE: Overriding `Web3Provider.ws_uri` implementation
         return "ws" + self.uri[4:]  # Remove `http` in default URI w/ `ws`
 
+    @property
+    def priority_fee(self) -> int:
+        if self.network.ecosystem.name == "polygon-zkevm":
```

Comment:
It appears that Alchemy is very limited with its RPCs for polygon zkevm for some reason.
Using a random node from evmchains works much better.

Reply:
ya they just support too many ETH RPCs